libp2p/Common.h

```diff
@@ -194,6 +194,15 @@ class NodeIPEndpoint
 	operator bool() const { return !address.is_unspecified() && udpPort > 0 && tcpPort > 0; }
 	
 	bool isAllowed() const { return NodeIPEndpoint::test_allowLocal ? !address.is_unspecified() : isPublicAddress(address); }
+
+	bool operator==(const NodeIPEndpoint rhs) const
```

Comment:
NodeIPEndpoint const& _cmp. Whitespace can be removed in header (single line function).